django/core/management/commands/shell.py

```diff
@@ -139,9 +140,20 @@ def get_auto_imports(self):
         ]
 
         """
+        try:
+            models = apps.get_models()
+        except AppRegistryNotReady:
+            if self.verbosity > 0:
+                self.stdout.write(
+                    "Models are not imported as not yet loaded. "
+                    "Check your settings are properly configured.",
```

Comment:
```suggestion
                    "Models are not imported because they are not "
                    "loaded yet. Check your settings are properly "
                    "configured and specified when running the "
                    "command.",
```

That may be too wordy now. I'm good with whatever you decide to go with though.